packages/fx-core/src/component/driver/teamsApp/interfaces/ApiSecretRegistration.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ApiSecretRegistrationClientSecret } from "./ApiSecretRegistrationClientSecret";
+
+export interface ApiSecretRegistration {
+  /**
+   * Max 128 characters
+   */
+  id: string;
+  /**
+   * Max 128 characters
+   */
+  description: string;
+  clientSecrets: ApiSecretRegistrationClientSecret[];
+  tenantId?: string;
+  /**
+   * Currently max length 1
+   */
+  targetUrlsShouldStartWith?: string[];
+  /**
+   * Teams app Id associated with the ApiSecretRegistration
+   */
+  specificAppId?: string;
+  applicableToApps?: ApiSecretRegistrationAppType[];
```

Comment:
According to [here](https://domoreexp.visualstudio.com/Teamspace/_git/teams-devapp?path=/Portal/src/interfaces/IApiKeyRegistration.ts), applicableToApps shouldn't be an array?